content/blog/tidymodels-2021-q1/index.Rmd

```diff
@@ -0,0 +1,141 @@
+---
+output: hugodown::hugo_document
+
+slug: tidymodels-2021-q1
+title: Catch up with tidymodels
+date: 2021-03-10
+author: Julia Silge
+description: >
+    Recent tidymodels releases offer new functions for easier model building and
+    resampling, along with a new package for resampling spatial data.
+
+photo:
+  url: https://unsplash.com/photos/4Zk45jNyQS4
+  author: Timo Wielink
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [package] 
+tags: [tidymodels, parsnip, rsample]
+---
+
+<!--
+TODO:
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
```

Comment:
Need to update this

Reply:
Is that something I do? I don't see that in the docs.

Reply:
I just delete them as I do them.

Reply:
Oh, I get what you mean -- just remove the instructions 🙈

Reply:
FWIW I thought you meant I should update the metadata